src/source/PeerConnection/PeerConnection.c

```diff
@@ -1442,6 +1602,45 @@ STATUS initKvsWebRtc(VOID)
     return retStatus;
 }
 
+STATUS cleanupWebRtcClientInstance()
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    // Stun object cleanup
+    PWebRtcClientContext pWebRtcClientContext = getWebRtcClientInstance();
+    CHK_WARN(ATOMIC_LOAD_BOOL(&pWebRtcClientContext->isContextInitialized), STATUS_INVALID_OPERATION,
```

Comment:
Need to release hold.

Reply:
Using an isReleased flag to ensure we release hold in case the client context is not initialized.